src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -447,7 +450,13 @@ public function checkOrderDifference(WC_Order $databaseOrder, $backofficeOrder):
 
     private function checkOrderDifferenceByExtra(array $databaseOrderItems, array $backofficeOrderItems): bool
     {
-        return array_column($databaseOrderItems, 'extra') !== array_column($backofficeOrderItems, 'extra');
+        $databaseOrderItemExtras = array_column($databaseOrderItems, 'extra');
+        $backofficeOrderItemExtras = array_column($backofficeOrderItems, 'extra');
+
+        sort($databaseOrderItemExtras);
+        sort($backofficeOrderItemExtras);
```

Comment:
Sorting array does not really work you need to sort by column, and also sort each array by key

```
php > $a = [['id' => 1,'name'=>'b'],['name'=>'a','id' => 2]];
php > $b = [['name'=>'a','id' => 2],['id' => 1,'name'=>'b']];
php > sort($a); sort($b);
php > var_export($a); var_export($b);
array (
  0 => 
  array (
    'id' => 1,
    'name' => 'b',
  ),
  1 => 
  array (
    'name' => 'a',
    'id' => 2,
  ),
)array (
  0 => 
  array (
    'name' => 'a',
    'id' => 2,
  ),
  1 => 
  array (
    'id' => 1,
    'name' => 'b',
  ),
)
php > var_export($a === $b);
false
```